### PR TITLE
Wdp191203 8

### PR DESCRIPTION
**Pull request by Oskyyy**
Using var's for colors in settings.scss

### CODE REVIEW

File: src/styles/settings.scss [hunk 1]
```diff
@@ -1,13 +1,42 @@
...
-
 $primary: #d58e32;
-
 $text-color: #2a2a2a;
-
 $form-border-color: #292929;
+
+/* colors used in app RIGHT NOW */
+$header-topbar-bg: $header-topbar-bg;
+$header-bg: $header-bg;
```
Comment (Varenthein) [inline src/styles/settings.scss]: What is that? :O

Comment (Oskyyy) [reply src/styles/settings.scss]: Good question. I don't know either but i changed it.

**Comment by Oskyyy** [comment]
It should work now. Surprisingly npm install helped.

**Comment by Varenthein** [comment]
Resolve conflicts please

**Comment by Oskyyy** [comment]
Conflict solved.

**Pull request merged**